templates/nextcloud28-entrypoint.sh

```diff
@@ -0,0 +1,125 @@
+mv /apps-to-install/nextcloud-social-login-*/* /var/www/html/custom_apps/sociallogin
+rm -r /apps-to-install/nextcloud-social-login-*
+rm /apps-to-install/*.zip
+
+chown -R www-data:www-data /var/www/html
+chmod -R 775 /var/www/html
+
+# Function from the Nextcloud's original entrypoint
+run_as() {
+    if [ "$(id -u)" = 0 ]; then
+        su -p www-data -s /bin/bash -c "$1"
+    else
+        sh -c "$1"
+    fi
+}
+
+run_as 'php occ config:system:set dbhost --value $MYSQL_HOST'
+run_as 'php occ app:enable sociallogin'
+
+OAUTH_SETTINGS=$(cat << DELIM
+'{"custom_oauth2":
+[
+{"name":"$OAUTH2_PROVIDER_NAME",
+"title":"$OAUTH2_PROVIDER_TITLE",
+"apiBaseUrl":"$OAUTH2_BASE_URL",
+"authorizeUrl":"$OAUTH2_AUTHORIZE_URL",
+"tokenUrl":"$OAUTH2_TOKEN_URL",
+"profileUrl":"$OAUTH2_PROFILE_URL",
+"logoutUrl":"$OAUTH2_LOGOUT_URL",
+"clientId":"$OAUTH2_CLIENT_ID",
+"clientSecret":"$OAUTH2_CLIENT_SECRET",
+"scope":"read",
+"profileFields":"$OAUTH2_PROFILE_FIELDS",
+"displayNameClaim":"$OAUTH2_NAME_FIELD",
+"groupsClaim":"$OAUTH2_GROUPS_CLAIM",
+"style":"",
+"defaultGroup":"",
+"groupMapping":{"$OAUTH2_ADMIN_GROUP":"admin"}}
+]}'
+DELIM
+)
+
+run_as "php occ config:app:set sociallogin custom_providers --value=$OAUTH_SETTINGS"
+
+
+# run_as 'php occ config:app:set sociallogin auto_create_groups --value=1'
+
+run_as 'php occ config:app:set sociallogin hide_default_login --value=1'
+run_as 'php occ config:app:set sociallogin update_profile_on_login --value=1'
+
+run_as 'php occ config:app:set settings profile_enabled_by_default --value=0'
+
+run_as 'php occ config:app:set dav generateBirthdayCalendar --value "no"'
+run_as 'php occ config:app:set dav sendEventReminders --value "no"'
+run_as 'php occ config:app:set dav sendReminders --value "no"'
+run_as 'php occ config:app:set dav sendInvitations --value "no"'
+
+run_as 'php occ config:app:set contacts allowSocialSync --value "no"'
+
+run_as 'php occ config:app:set core shareapi_allow_links --value="no"'
+run_as 'php occ config:app:set core shareapi_allow_resharing --value="no"'
+run_as 'php occ config:app:set core shareapi_allow_share_dialog_user_enumeration --value="no"'
+run_as 'php occ config:app:set core shareapi_restrict_user_enumeration_full_match --value="no"'
+
+run_as 'php occ config:app:set files_sharing incoming_server2server_share_enabled --value="no"'
+run_as 'php occ config:app:set files_sharing lookupServerEnabled --value="no"'
+run_as 'php occ config:app:set files_sharing lookupServerUploadEnabled --value="no"'
+run_as 'php occ config:app:set files_sharing outgoing_server2server_share_enabled --value="no"'
+
+run_as 'php occ config:system:set social_login_auto_redirect --value=true'
+run_as 'php occ config:system:set skeletondirectory'
+run_as "php occ config:app:set sociallogin custom_providers --value=$OAUTH_SETTINGS"
+
+run_as 'php occ app:install calendar'
+run_as 'php occ app:install polls'
+run_as 'php occ app:install contacts'
+run_as 'php occ app:install richdocumentscode'
```

Comment:
remove & uninstall onlyoffice and onlyoffice code stuff - we only want collabora